apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -896,9 +902,23 @@ events:
         type: Dictionary
 
       - name: inBackground
-        summary: Boolean indicating if notification was received while app was in background (since Titanium SDK 6.2.0).
+        summary: |
+          Boolean indicating if notification was received while app was in background (since Titanium SDK 6.2.0).
+          On iOS 10+ this is no longer available since the event will only fire when opening the app after interacting
+          with a notification from the iOS notification center (which means the app was in background when the
+          notification was received).
         type: Boolean
 
+      - name: threadIdentifier
+        summary: |
+          The unique identifier for the thread or conversation related to this notification request.
+          It will be used to visually group notifications together. Available in Titanium SDK 7.4.0+ and iOS 10+.
```

Comment:
Since this is a new property, shouldn't this be `Available in Titanium SDK 7.5.0+`

Also, shouldn't we be doing:
```
since: { ios: "7.5.0" }
```